stock_buffer_capacity_limit/models/stock_buffer.py

```diff
@@ -21,10 +22,18 @@ def _compute_procure_recommended_qty(self):
         for rec in self.filtered(lambda b: b.storage_capacity_limit > 0.0):
             recommendation_limit = max(
                 rec.storage_capacity_limit
-                - rec.product_id.qty_available
+                - rec.product_id.with_context(location=rec.location_id.id).qty_available
```

Comment:
This is not returning the qty of the location. This solved it:
```suggestion
                - rec.product_location_qty
```
I don't know why, but `rec.product_id.with_context(location=rec.location_id.id).qty_available` does not trigger the computation but reads the result from cache and that contains the qty available of the product, not of the location

Reply:
That's interesting, sometimes the new v13 cache has this surprises for us. I'm commiting your proposal now.